RecoTracker/MkFit/python/dumpMkFitGeometry.py

```diff
@@ -0,0 +1,48 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_cff import Run3
+from Configuration.ProcessModifiers.trackingMkFit_cff import trackingMkFit
+
+process = cms.Process('DUMP',Run3,trackingMkFit)
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+# process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
```

Comment:
I'd remove all these unnecessary loads instead of commenting out.

Reply:
removed